pkg/app/pipedv1/plugin/azure/config/config.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+type AzureDeployTargetConfig struct {
+	SubscriptionID string `json:"subscriptionID"`
+	//shared key file, env file, env variable?
+}
+
+type AzureApplicationSpec struct {
+	Kind             string         `json:"kind"`
+	FunctionManifest *FunctionsSpec `json:"functionManifest"`
```

Comment:
'Kind' is confusing in K8s-like yamls.

Instead of introducing a new word 'FunctionManifest', using 'Input' is simpler for users.

```suggestion
	Type             string         `json:"type"`
	Input *FunctionsDeploymentInput `json:"input"`
```

See also:
https://github.com/pipe-cd/pipecd/blob/master/pkg/config/application_kubernetes.go#L18-L22